provider/datagen/src/driver.rs

```diff
@@ -735,12 +733,12 @@ impl DatagenDriver {
                     }
                     Some(x) => x,
                 };
-                let mut sorted_locales = families
+                let mut sorted_locale_strs = families
                     .iter()
                     .map(LocaleFamilyBorrowed::from_parts)
                     .map(|family| family.write_to_string().into_owned())
                     .collect::<Vec<_>>();
-                sorted_locales.sort_unstable();
+                sorted_locale_strs.sort_unstable();
                 log::info!(
                     "Datagen configured with {}, {}, and these locales: {:?}",
```

Comment:
yeah bit weird that these are `LanguageIdentifiers` on the API, but `locales` in logs/CLI.